plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoOutputTableHandle.java

```diff
@@ -28,14 +29,20 @@
 {
     private final SchemaTableName schemaTableName;
     private final List<MongoColumnHandle> columns;
+    private final Optional<String> temporaryTableName;
+    private final Optional<String> pageSinkIdColumnName;
 
     @JsonCreator
     public MongoOutputTableHandle(
             @JsonProperty("schemaTableName") SchemaTableName schemaTableName,
-            @JsonProperty("columns") List<MongoColumnHandle> columns)
+            @JsonProperty("columns") List<MongoColumnHandle> columns,
+            @JsonProperty("temporaryTableName") Optional<String> temporaryTableName,
+            @JsonProperty("pageSinkIdColumnName") Optional<String> pageSinkIdColumnName)
     {
         this.schemaTableName = requireNonNull(schemaTableName, "schemaTableName is null");
         this.columns = ImmutableList.copyOf(requireNonNull(columns, "columns is null"));
+        this.temporaryTableName = requireNonNull(temporaryTableName, "temporaryTableName is null");
+        this.pageSinkIdColumnName = requireNonNull(pageSinkIdColumnName, "pageSinkIdColumnName is null");
```

Comment:
verify either both or none set. Those always come in pairs, right?

Reply:
gah - i added that to MongoInsertTableHandle and forgot to here.